includes/islandora_scan_batch.inc

```diff
@@ -520,6 +520,20 @@ EOXML;
 
     $this->relationships->add($uri, $predicate, $collection);
     $this->addContentModelRelationships();
+    $this->inheritXacmlPolicies();
+  }
+
+  /**
+   * Add any XACML policies from the parents.
```

Comment:
Comment seems misleading... We don't add the `POLICY` directly, we just write the relationship so it should happen later... Not sure what an ideal comment would look like...